README.md

```diff
@@ -76,3 +87,44 @@ The following exercises are optional, but we suggest that you tackle them if you
   - add a `DELETE /games/:id` endpoint that can remove the corresponding game. If the game does not exists return a 404 status code. Write tests for this endpoint.
 
 **Remember you can use any resources you want to solve these problems, but avoid copying/pasting solutions you've previously written. Also if you don't finish all of the challenges, that's fine! Just do what you can and submit your challenges in the end! HAVE FUN!**
+
+
+
+[
+    {
+        title: 'Asteroids',
+        genre: 'Shooter',
```

Comment:
this object can be placed in your testing file and be used as the data instead of migrations and seeding